devito/finite_differences/differentiable.py

```diff
@@ -288,6 +288,15 @@ def subs(self, *args, **kwargs):
         return self.func(*[getattr(a, 'subs', lambda x: a)(*args, **kwargs)
```

Comment:
so this one doesn't call `_subs` directly, but, I'd guess, only via the superclass? that's a bit odd? or maybe not, actually

Reply:
Need subs for when there is more than one arg. `_subs` only accept `u._subs(old, new)` for dict/tuple of tuple this one is used. This is still a bit more efficient than the default sympy.expr version of it.